CHANGES/7014.bugfix

```diff
@@ -0,0 +1 @@
+Fixed bug when using TCPConnector with ttl_dns_cache=0.
```

Comment:
```suggestion
Fixed bug when using ``TCPConnector`` with ``ttl_dns_cache=0``.
```

Think that'll fix the spell check problem.

Reply:
Thanks for the hint. :crossed_fingers: